stack-snapshot.yaml

```diff
@@ -3,10 +3,10 @@
 
 resolver: nightly-2021-01-05
 packages:
-  - archive: https://daml-binaries.da-ext.net/da-ghc-lib/ghc-lib-927591afb5343097516894c6163a6df7.tar.gz
-    sha256: "7014dd6b9b277ecc6c41de8331eeb741c8f255aea784414b43aa668e98bef5d7"
-  - archive: https://daml-binaries.da-ext.net/da-ghc-lib/ghc-lib-parser-927591afb5343097516894c6163a6df7.tar.gz
-    sha256: "5873fdd7ab9c9133c658bcc29884f4eaba0f40785caa157e1ba93809a3f4a651"
+  - archive: https://daml-binaries.da-ext.net/da-ghc-lib/ghc-lib-7382eb366ad3cf839304c1b640f870c9.tar.gz
```

Comment:
let’s bump this to a the commit after merging the GHC pr.

Reply:
Yep! Waiting for the GHC PR to be merged before doing another round of ghc-lib updates (and including windows).